upi/openstack/network.yaml

```diff
@@ -87,18 +89,18 @@
       - subnet: "{{ os_subnet }}"
         ip_address: "{{ os_ingressVIP }}"
     register: _ingress_ports
-    when: os_subnet6 is not defined
+    when: os_subnet6_range is not defined
 
   - set_fact:
       ingress_ports: "{{ _ingress_ports }}"
-    when: _ingress_ports.changed
+    when: _api_ports.changed
```

Comment:
This change seems wrong. Is it intentional?

Reply:
no, it's not.